### PR TITLE
Refactor/modules and add stats module

### PR DESCRIPTION
**Pull request by rafinskipg**
### What does this PR do?

- Adds the /api/stats/address endpoint that will return the votes for a certain address on an API level.
- Refactors old code into modules, (not all of it, is a progressive approach). This is just the /lib/api file that contained logic for different things, i splitted that into modules and in future PRs we can migrate more things from lib

See RFC #98 for module structure

image

### CODE REVIEW

File: components/header/VotingWeight.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Flex, Text } from 'theme-ui';
 import useSWR from 'swr';
 import useAccountsStore from 'stores/accounts';
 import getMaker from 'lib/maker';
-import { getVotingWeightCopy } from 'lib/polling/getVotingWeightCopy';
+import { getVotingWeightCopy } from 'modules/polls/lib/getVotingWeightCopy';
```
Comment (adamgoth) [inline components/header/VotingWeight.tsx:5]: I don't have very strong feelings about this but I feel like `utils` or `helpers` could be a better directory name for `lib` within these modules. The functions inside seem mostly to be utility/helper functions. Any thoughts?

File: modules/blog/api/fetchBlogPosts.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { BlogPost } from '../types/blogPost';
+
+
+export async function fetchBlogPosts(): Promise<BlogPost[]> {
+  // these are not just mocks anymore, but the actual blog posts that should be shown in production
+  return require('./mocks/blogPosts.json');
```
Comment (adamgoth) [inline modules/blog/api/fetchBlogPosts.ts:6]: I know you didn't write this, just moved the code, but we should probably update this so our actual blog posts aren't coming from a mocks file 🤔

File: modules/stats/api/fetchStats.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    votes: [
+      {
+        date: Date.now(),
+        address: '',
+        value: 'yes'
+      }
+    ]
+  });
+}
```
Comment (adamgoth) [inline modules/stats/api/fetchStats.ts:23]: Maybe add a comment that this is a placeholder for now? The actual API hasn't been figured out yet.

**Review by adamgoth — APPROVED** [review]
I dig it in general. I left a few thoughts.

File: modules/blog/api/fetchBlogPosts.ts (hunk 2 above)
Comment (rafinskipg) [reply modules/blog/api/fetchBlogPosts.ts]: done

**Pull request merged**